### PR TITLE
refactor: refactored loginUser to return the user object

### LINKED ISSUE #130: Return logged user in login http layer
We should return username, email and account type instead of 'true' to frontend in loginUser function after successful login

### PR DESCRIPTION
**Pull request by kpagacz**
* also added basic documentation to the function
Closes #130

I tested it with the following code:
```js
import { loginUser } from "./src/httpLayers/login.http.js";

console.dir(await loginUser("test_username1", "test_password"));
```

### CODE REVIEW

**Comment by kpagacz** [comment]
@karolkozlo let me know if you want me to formalize the user object

File: frontend/src/httpLayers/login.http.js [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
 
+/**
+ * Authenticates the user to the backend service of
+ * WikiOnFire. The response from the server sets
+ * the authentication and authorization `httpOnly` cookie.
+ *
+ * @param {String} username
+ * @param {String} password
+ * @returns {Object} the user object with the following fields:
+ *   id, username, password, email, account_type, account_status, createdAt,
```
Comment (karolkozlo) [inline frontend/src/httpLayers/login.http.js:11]: We should not return password here

Comment (kpagacz) [reply frontend/src/httpLayers/login.http.js:11]: Mhm, I am going to have a look at the endpoint as well. It shouldn't return password as well.

Comment (kpagacz) [reply frontend/src/httpLayers/login.http.js]: Fixed

**Review by karolkozlo — APPROVED** [review]

**Pull request merged**